content/en/docs/use-cases/running-jobs.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Running pre and post-deployment jobs with Flux
+linkTitle: Running Kubernetes Jobs with Flux
+description: "How to run Kubernetes Jobs before and after an application deployment with Flux."
+weight: 10
```

Comment:
Not sure but I think this value determines the order of the page at https://deploy-preview-990--fluxcd.netlify.app/docs/use-cases/ and I'm not sure how the ordering is supposed to look like. @stefanprodan might know. Maybe we just don't care.

Reply:
I will fix the ordering and titles of all guides under use-cases in a a different PR. Thanks for bringing it up, I wanted to to this long time ago and I forgot about it.